plugins/orchestrator/package.json

```
@@ -28,14 +28,16 @@
   ],
   "scripts": {
     "start": "backstage-cli package start",
-    "build": "backstage-cli package build",
+    "build": "yarn openapi:check && backstage-cli package build",
     "export-dynamic": "janus-cli package export-dynamic-plugin",
     "tsc": "tsc",
     "lint": "backstage-cli package lint",
     "test": "backstage-cli package test --passWithNoTests --coverage",
     "clean": "backstage-cli package clean",
     "prepack": "backstage-cli package prepack",
-    "postpack": "backstage-cli package postpack"
+    "postpack": "backstage-cli package postpack",
+    "openapi:generate:client": "./scripts/openapi.sh generate",
```

```suggestion
    "generate": "./scripts/openapi.sh generate",
```

To maintain consistency, I believe it's beneficial to use the same script name as other plugins.

---

@gciavarrini 
My instinct when cloning the repo and trying to use it is to go to the backend to generate the client, since this isn't an operation related to the UI
IIUC the purpose is to enable the user to consume the backend easily.
Do you think it can move to the backend?

---

I currently have two different tasks: `openapi:generate:client` and `openapi:check`. 
While I agree that consistency is beneficial, using names like `generate` and `check` might be too generic and could lead to confusion. Perhaps a good compromise would be to rename them to `openapi:generate` and `openapi:check` to simplify the names while still maintaining clarity. What do you think?

---

Would you be willing to update the other `generate` script references to `openapi:generate` in the repository in a separate PR?

---

Do you mean in other plugins as well (e.g. `servicenow-actions`)?  If so, sure, I will update those in a separate PR

---

Open PR https://github.com/janus-idp/backstage-plugins/pull/1708